### ISSUE TITLE
addition of a new page under kolkata

### ISSUE THREAD

**Issue by GhoshAditi** [jaysomani/firstspot#93]
addition of page about history and tourist attraction about kolkata . this page will open when the user clicks on the picture of kolkata under places of visit. This will make it more convenient for the user to decide where to go on basis of his preference.
@jaysomani please assign it to e under swoc'24

**Comment by GhoshAditi** [comment]
@jaysomani i would like to work on it and this time create page for the website only

**Comment by jaysomani** [comment]
Hey @GhoshAditi you can work on this and keep me updated in every 3 days

**Comment by GhoshAditi** [comment]
@jaysomani yes working on it. Almost finished it as well. Just some more adjustments needed to be done. Hope I can complete it within a couple of days.
